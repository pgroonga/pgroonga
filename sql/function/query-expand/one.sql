SELECT pgroonga.query_expand('Groonga');
