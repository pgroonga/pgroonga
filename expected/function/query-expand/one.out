SELECT pgroonga.query_expand('Groonga');
 query_expand 
--------------
 Groonga
(1 row)

